Rocket.Chat/API/Requests/Subscription/SubscriptionCreateRequest.swift

```diff
@@ -62,4 +62,11 @@ extension APIResult where T == SubscriptionCreateRequest {
     var error: String? {
         return raw?["error"].string
     }
+
+    var name: String? {
+	    guard let group = raw?["group"], group["t"] == "p" else {
+            return raw?["channel"]["name"].string
+		}
+        return raw?["group"]["name"].string
```

Comment:
Nice! 👍

Reply:
Looks like I meessed up the indentation here, a new commit would be good, @cardoso?

Reply:
Yes @soundstorm, please fix that.

Reply:
Done :)

Reply:
Just switched my Xcode to 4 spaces instead of tabs again, so this shouldn't be a problem anymore